cloudinary_cli/modules/upload_dir.py

```diff
@@ -23,20 +23,29 @@
 @option("-f", "--folder", default="",
         help="The Cloudinary folder where you want to upload the assets. "
              "You can specify a whole path, for example folder1/folder2/folder3. "
-             "Any folders that do not exist are automatically created.")
+             "If your product environment uses fixed folder mode, then any folders that do not exist are automatically created.")
 @option("-p", "--preset", help="The upload preset to use.")
+@option("-e", "--exclude-dir-name", is_flag=True, default=False,
+        help="Don't include the selected parent directory name in the public ID path of the uploaded files."
+             "This ensures that the public ID paths of the uploaded assets will be directly under the specified --(f)older, avoiding an extraneous level in the path."
+             "When this option is used, the contents of the parent directory are uploaded instead of the parent directory itself and thus the name of the specified parent directory is not included in the pubic ID path of the uploaded assets.)
```

Comment:
```suggestion
             "When this option is used, the contents of the parent directory are uploaded instead of the parent directory itself and thus the name of the specified parent directory is not included in the pubic ID path of the uploaded assets.")
```